subscription_manager/utils.py

```diff
@@ -1,7 +1,56 @@
-from typing import List
+from dataclasses import fields
+from typing import Tuple
+from random import choice, uniform, randint
+from datetime import datetime, date
+from subscription_manager.exceptions import *
+from subscription_manager.subscription import Subscription
 
 
-def _validate_subscription(subscription: dict) -> bool:
+def _get_currencies_list() -> Tuple[str]:
```

Comment:
please use constance.py and set 

CURRENCIES = ('USD', 'GBP', 'EUR', 'RUB', 'CNY')

Reply:
done